DevoxxRemoteFunctions/build.gradle

```diff
@@ -1,4 +1,26 @@
+import jp.classmethod.aws.gradle.lambda.AWSLambdaUpdateFunctionCodeTask
+
+buildscript {
+    repositories {
+        mavenCentral()
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+    dependencies {
+        classpath "jp.classmethod.aws:gradle-aws-plugin:0.+"
+    }
+}
+
 apply plugin: 'java'
+apply plugin: 'jp.classmethod.aws.lambda'
+
+aws {
+    profileName = "default"
+    region = "eu-west-1"
+}
+
+lambda {
+    region = "eu-west-1"
```

Comment:
If the region is the same as the one that is specified in the global aws configuration block above, I suppose you can omit this.